src/app/[locale]/admin/revenue/components/RevenueFilter.tsx

```
@@ -33,40 +48,155 @@ interface RevenueFilterProps {
+
+    const handleMonthlyQuickFilter = (preset: 'thisMonth' | 'lastMonth' | 'last6Months' | 'ytd') => {
+        const today = startOfToday();
+        let from: Date;
+        let to: Date = today;
+
+        switch (preset) {
+            case 'thisMonth':
+                from = startOfMonth(today);
+                break;
+            case 'lastMonth':
+                const lastMonth = subMonths(today, 1);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)